src/core/transaction.c

```diff
@@ -253,7 +254,26 @@ static int transaction_merge_jobs(Transaction *tr, sd_bus_error *e) {
                 }
         }
 
-        /* Second step, merge the jobs. */
+        return 0;
+}
+
+static int transaction_merge_jobs(Transaction *tr, sd_bus_error *e) {
+        Job *j;
+        int r;
+
+        assert(tr);
+
+        /* First step, try to drop unmergable jobs for anchor jobs. */
```

Comment:
anchor jobs -> jobs that matter to anchor

Reply:
Thanks. Will update.